aura/lib/Aura/Settings/External.hs

```diff
@@ -45,17 +46,28 @@ data AuraConfig = AuraConfig
   , acAnalyse   :: !(Maybe BuildSwitch) }
   deriving stock (Show)
 
-defaultAuraConf :: FilePath
-defaultAuraConf = "/etc/aura.conf"
+userAuraConfPath :: IO FilePath
+userAuraConfPath = getUserConfigFile "aura" "config"
```

Comment:
Can we call the file itself `aura.conf`? This will mean fewer necessary changes for documentation elsewhere.

Reply:
I've had a quick search and can't find anything on standardisation of filenames. I've noticed also that some other projects like mpd follow this naming convention (`app/app.conf`). :+1: